src/dotnet-svcutil/lib/src/Shared/ProjectDependency.cs

```diff
@@ -28,6 +28,7 @@ internal class ProjectDependency : IComparable<ProjectDependency>
 
         private static readonly IReadOnlyList<string> s_binaryExtensions = new List<string> { ".exe", ".dll" }.AsReadOnly();
```

Comment:
is this private member still needed?  or is it possible to modify this with .exe only?

Reply:
Yes, it's being used in other code path, e.g. line115, L133, L315.